Resources/Prototypes/Entities/Objects/Weapons/Throwable/grenades.yml

```diff
@@ -88,6 +88,115 @@
       countdown_sound:
         path: /Audio/Effects/minibombcountdown.ogg
 
+- type: entity
+  name: supermatter grenade
+  description: Grenade that simulates delamination of supermatter engine, pulling things in heap and explode after some time.
+  parent: GrenadeBase
+  id: SupermatterGrenade
+  components:
+  - type: Sprite
+    sprite: Objects/Weapons/Grenades/supermattergrenade.rsi
+  - type: OnUseTimerTrigger
+    delay: 3
+  - type: PointLight
+    enabled: false
+    color: "#ffaa44"
+    energy: 8
+    radius: 6
+    softness: 1
+    offset: "0, 0"
+  - type: Explosive
+    explosionType: Default
+    totalIntensity: 200
+    intensitySlope: 30
+    maxIntensity: 120
+  - type: Appearance
+    visuals:
+    - type: TimerTriggerVisualizer
+      countdown_sound:
+        path: /Audio/Effects/minibombcountdown.ogg
+  - type: SingularityDistortion
+    intensity: 0
+    falloffPower: 2
+  - type: GravityWell
+    maxRange: 8
+    baseRadialAcceleration: 0
+    gravPulsePeriod: 0.01
+  - type: SoundOnTrigger
+    removeOnTrigger: true
+    sound:
+      path: /Audio/Effects/Grenades/Supermatter/supermatter_start.ogg
+      volume: 5
+  - type: AmbientSound
+    enabled: False
+    volume: -5
+    range: 14
+    sound:
+      path: /Audio/Effects/Grenades/Supermatter/supermatter_loop.ogg
+  - type: AnchorOnTrigger
+    removeOnTrigger: true
+  - type: SingularityDistortionOnTrigger
+    removeOnTrigger: true
+    intensity: 100 
+  - type: GravityWellOnTrigger
+    removeOnTrigger: true
+    radialAcceleration: 145
+  - type: PointLightEnableOnTrigger
+    removeOnTrigger: true
+  - type: TwoStagedGrenade
+    explodeAfterGravityPull: True
+    explossionDelay: 13.2
+    ambienceSoundOffset: 0.4
+    # componentsToAdd:
+    #   - ExplodeOnTriggerComponent
+    #   - AmbientSoundComponent
+    #     enabled: False
+    #     volume: -5
+    #     range: 14
+    #     sound:
+    #       path: /Audio/Effects/Grenades/supermatter_loop.ogg
+    #   - SoundOnTriggerComponent
+    #      sound:
+    #        path: /Audio/Effects/Grenades/supermatter_end.ogg
+    #        params:
+    #          volume: 5
+    endSound:
+      path: /Audio/Effects/Grenades/Supermatter/supermatter_end.ogg
+      params:
+        volume: 5
+
+- type: entity
+  name: whitehole grenade
+  description: Grenade that are repulses everything around for sometime. Very useful in close rooms and for chasing someone.
```

Comment:
remove are
a round -> around
sometime -> some time
close -> small or tight
and -> or